### PR TITLE
feat(engine): set_replay_file during runtime

### PR DESCRIPTION
**Pull request by zhc134**

### CODE REVIEW

**Comment by ThisIsIsaac** [comment]
I can't `git clone` your branch with commit[8478f61ace910586747e76fd403130dcfbddb5e0](https://github.com/cityflow-project/CityFlow/pull/31/commits/8478f61ace910586747e76fd403130dcfbddb5e0) for some reason. When I git clone by:

`git clone --recursive https://github.com/zhc134/CityFlow.git`

and check `git log`, the last two commits are:

```
commit 7f2cc2e1a31bcb2a8592ae2a3f54ddd02bae9e9d
Author: zhc134 <44251346+zhc134@users.noreply.github.com>
Date:   Thu Aug 29 15:55:01 2019 +0800

    docs: eng.reset() and replay (#30)

commit 04b36f62c14c6386648f918f3f0386a8236b65b5
Author: 이중건 Isaac Lee <49523965+ThisIsIsaac@users.noreply.github.com>
Date:   Thu Aug 29 16:07:34 2019 +0900

    fix(UI): support path starting with '/' (#28)
```

as you can see from the log, the commit that implements `set_log_file` is not included.

But when I download it as a zip file, the last commit that was missing is properly applied. I've never seen this kind of issue before. Do you think you have a clue?

**Comment by zhc134** [comment]
@ThisIsIsaac 
This feature is developed in a separate branch, try checkout `issue-29` branch?

**Comment by ThisIsIsaac** [comment]
When I do `git branch` I only see `master`.

And I got the address from your branch:
Screen Shot 2019-09-04 at 16 14 56 PM

which is: `https://github.com/zhc134/CityFlow.git`

**Comment by zhc134** [comment]
Try
```
git fetch
git checkout issue-29
```

**Comment by ThisIsIsaac** [comment]
worked like charm! thanks!

**Comment by ThisIsIsaac** [comment]
I have successfully installed from `issue-29` but I get:

```
>>> import cityflow
>>> cityflow.set_replay_file("hello.txt")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'cityflow' has no attribute 'set_replay_file'
```

from python 3.6.8

**Comment by zhc134** [comment]
the API is bind to engine object.
try
```
eng = cityflow.Engine(config, 1)
eng.set_replay_file("path")
```

File: tests/python/test_api.py [hunk 1]
```diff
@@ -26,6 +26,19 @@ def test_data_api(self):
...
+        eng = cityflow.Engine(config_file = self.config_file, thread_num = 1)
+
+        for _ in range(100):
+            eng.next_step()
+        
+        eng.set_replay_file("replay2.txt")
+
+        for _ in range(100):
+            eng.next_step()
+
```
Comment (Hzfengsy) [inline tests/python/test_api.py]: Please check PEP8 style

**Pull request merged**